tests/core/test_config.py

```diff
@@ -98,3 +101,19 @@ def load_cdns(cls, _db, config_instance=None):
         assert new_db != self._db
         assert isinstance(new_db, Session)
         assert None == none
+
+    @patch.object(os, "environ", new=dict())
```

Comment:
Another option that `pytest` gives us `monkeypatch.setenv` (and `monkeypatch.delenv`)... https://docs.pytest.org/en/latest/reference/reference.html#pytest.MonkeyPatch.setenv, but this should work, too.

Reply:
I generally like to stick to builtin unit-testing where available